React/tic-tac/src/app/screens/Login/components/styles.scss

```diff
@@ -0,0 +1,87 @@
+.inputField {
```

Comment:
Try not to set component styles outside of the component folder since it will make reusability harder. 🤓